Tests/eppo/UniversalFlagConfigTests.swift

```diff
@@ -0,0 +1,47 @@
+import XCTest
+
+import Foundation
+
+@testable import eppo_flagging
+
+final class UniversalFlagConfigTest: XCTestCase {
+    var fileURL: URL!
+    var UFCTestJSON: String!
+
+    override func setUp() {
+        super.setUp()
+        fileURL = Bundle.module.url(
+            forResource: "Resources/test-data/ufc/flags-v1.json",
+            withExtension: ""
+        )
+        do {
+            UFCTestJSON = try String(contentsOfFile: fileURL.path)
+        } catch {
+            XCTFail("Error loading test JSON: \(error)")
+        }
+    }
+
+    func testDecodeUFCConfig() {
```

Comment:
I'm not trying to be exhaustive here just touch some of the models to make sure they are decoding as expected. We need to count on the json parsed throwing an error to truly detect mismatches.

Reply:
Makes sense, ideally we can count on the general test cases to provide good testing coverage

Reply:
Yes I believe the test files exercise all permutations in one way or another: some have no rules, variety of splits, etc. I implemented off the UFC API spec instead of the test files so that should cover all cases. Going to gRPC will be even better 🤓